site/content/docs/5.2/components/list-group.md

```diff
@@ -102,6 +102,11 @@ Add the `.list-group-numbered` modifier class (and optionally use an `<ol>` elem
 
 Numbers are generated by `counter-reset` on the `<ol>`, and then styled and placed with a `::before` pseudo-element on the `<li>` with `counter-increment` and `content`.
 
+<!-- Boosted mod : design callout -->
```

Comment:
Some explanations 😄 : 
- the 1st numbered list would be OK with a few design corrections.
- the 2nd numbered list would be OK with a few design corrections and without the badge and/or by replacing it by a status Icon such as ![image](https://user-images.githubusercontent.com/89514509/202716276-aa40d106-a21d-4783-87be-da6758041a1d.png)

Reply:
ON HOLD:  We have a PR to correct the design of the variants => delete the present callout.

The only one that will need a callout will be the variant with the badges. Its text:
This variant should not be used because it displays badges that do not respect the Orange Design System specifications.

Reply:
Agree. This callout must be deleted.

Reply:
DONE